0. Kirish/0.1 Maqsad, Modellashtirish va Simulyatsiya.md

```diff
@@ -0,0 +1,22 @@
+# 0.1 Maqsad, Modellashtirish va Simulyatsiya
+
+## 0.1.0 Kirish
+
+"Meyor" dasturiy ta’minot tuzilishini uch asosiy yo'nalishda ifodalaydi: Maqsad, Modellashtirish va Simulyatsiya. Ularning har biri muhandislarning muvaffaqiyatli yechim va aniq maqsadga erishishlarida muhim rol o'ynashdi. Bu yo'nalishlarning bajarilish ketma-ketligi ham o'zaro bog'liqdir. Maqsad Modellashtirish jarayonida mavjud bo'lishi shart. Ayni paytda Simulyatsiya jariyoni ham modellarsiz bajarila olmaydi. Bunday ketma-ketlik muhandislik faoliyati boshlanishida hal qiluvchi ma’no kasb etsadi, shuni anglash kerakki, jarayonning o'zi tanlab takrorlanadi. Maqsaddagi o'zgarish Simulyatsiyada o'z aksini topishi mumkin, ammo Modelni o'zgartirish shart bo'lmaydi. Modeldagi o'zgarish na Maqsad va na Simulyatsiyani o'zgartirishini taqazo qilishi mumkin.
+
+<br />
+    <div align=center>
+        <img src="./Resurslar/Maqsad, Modellashtirish va Simulyatsiya/maqsad,modellashtirish va simulyatsiya.png" />
+    </div>
+<br />
+
+
+## 0.1.1 Maqsad
+
+Maqsad qo'yish jarayoni - nima uchun bizga yechim kerakligini aniqlash qobiliyatidir. Misol uchun, ma'lum bir oziq-ovqat do'kon javonida qancha mahsulot borligini bilish bilan bog'liq muammo bor. Biz qo'lda hisoblash jarayonini samarasiz deb hisoblaymiz va mahsulotlarni to'g'ri hisoblashimiz uchun tizimni joriy qilish kerak.
+
+Mulohaza yuritish ko'p jihatdan muammolarni kuzatish va keyin ushbu muammoni hal qiladigan yechimni ishlab chiqish uchun muammoni ifodalash qobiliyatiga bog'liq.
+
+Shunday qilib, biz muommoni kuzatish, fikrlash va uni hal qilishimiz mumkun.Bu jihatlarning barchasi muhandislik dasturiy ta'minotining maqsadli qismini tashkil qiladi.
```

Comment:
`So, we have the observation, the articulation of the reasoning (the problem), and the intent for a solution.` =>

Man bu gapni quyidagicha tushundim:
`Shunday qilib, biz kuzatdik, muammoni mohiyatini tushundik va yechimni ishlab chiqish uchun maqsadga ega bo'ldik.` 

Agar sizga ham shu taklif ma'qul kelgan bo'lsa, shu gapni o'zgartirishni o'ylab ko'ring

Reply:
`Bu jihatlarning barchasi muhandislik dasturiy ta'minotining Maqsadlashtirish qismini tashkil qiladi.`

Reply:
ok chunarli